CHANGES/7195.bugfix

```diff
@@ -0,0 +1 @@
+Made UpdateCollection be unique on (name, update_record).
```

Comment:
It's a non-user-facing change, so I suggest to make it .misc.

Reply:
done